src/quo2/core.cljs

```diff
@@ -303,3 +304,5 @@
 (def account-card quo2.components.wallet.account-card.view/view)
 (def token-input quo2.components.wallet.token-input.view/view)
 
+;;;; SWITCHER
```

Comment:
Let's keep these Alpha sorted for category too 👍 
This file is huge and easier to keep organised this way